cassandra/src/test/scala/akka/stream/alpakka/cassandra/scaladsl/README.txt

```diff
@@ -0,0 +1,7 @@
+All the tests must be run with a local Cassandra running on default port 9042.
+
+When using docker, a temporary server can by started by running:
+
+```
+docker run --rm -p 9042:9042 cassandra:3
+```
```

Comment:
I agree. One might not be coming from the docs. What is more, this information is only relevant when someone tries to run the tests. In a light of this realization I think it would be best to have this information printed out to the console in `beforeAll` methods of the spec (and java test).

This might be an unorthodox suggestion, be we should not be reserved when it comes to communication through sbt console. :) P.S. [Scala has it](https://github.com/scala/scala/blob/2.12.x/build.sbt#L859).